readthedocs/builds/querysets.py

```diff
@@ -93,9 +112,27 @@ def _add_from_user_projects(self, queryset, user, admin=False, member=False):
         return queryset
 
     def public(self, user=None, project=None):
-        queryset = self.filter(
-            version__privacy_level=constants.PUBLIC,
-            version__project__privacy_level=constants.PUBLIC,
+        """
+        Get all allowed builds.
+
+        Builds are public if the linked version and project are public.
+
+        .. note::
+
+           External versions use the `Project.external_builds_privacy_level`
+           field instead of its `privacy_level` field.
```

Comment:
```suggestion
           NOTE: External versions use the `Project.external_builds_privacy_level`
           field instead of its `privacy_level` field.
```

Not sure these need to be RST notes..

Reply:
They render nicely in the editor :D

Reply:
Which editor?

Reply:
neovim :wink: 

![Screenshot from 2021-06-24 19-28-13](https://user-images.githubusercontent.com/4975310/123351292-68df4880-d522-11eb-942a-535fb02b9f2c.png)